ironfish-cli/src/commands/status.ts

```diff
@@ -219,7 +222,7 @@ function renderStatus(content: GetNodeStatusResponse, debugOutput: boolean): str
     Version: `${content.node.version} @ ${content.node.git}`,
     Node: nodeStatus,
     'Node Name': content.node.nodeName,
-    'Peed ID': content.peerNetwork.publicIdentity,
+    'Peer ID': content.peerNetwork.publicIdentity,
```

Comment:
🤦 i cant believe ive done this https://github.com/iron-fish/ironfish/blame/master/ironfish-cli/src/commands/status.ts#L222